src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets.utils;
+package com.google.sps.servlets;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Statistics {
   // Aggregated score for a list of comments  
-  // Contains sentiment scores in the range [-1, 1] with 0.2 intervals.
-  private List<Double> aggregateValues;
+  // Contains number of sentiment scores in the range [-1, 1] with 0.2 intervals.
+  private List<Integer> aggregateValues;
   private double averageScore;
 
-  public Statistics(List<Double> aggregateValues, double averageScore) {
+  public Statistics(List<Integer> aggregateValues, double averageScore) {
     this.aggregateValues = aggregateValues;
```

Comment:
With this current design, the creation of a Statistics object requires consumers to perform their statistical analsyis first.

Ideally, this statistical analysis should be baked into the Statistics object itself. 

This kind of pattern is usually conducive to static factory methods. (https://en.wikipedia.org/wiki/Factory_method_pattern)

I propose we add a new method like:

```
public static Statistics fromCommentList(List<Comment> commentList){
  // Perform analysis and create + return Statistics object.
}
```

In doing so, we should also make the Statistics constructor private since objects of this class would only need to be constructed within this class. 

Note that this method should likely come in a follow-up PR.

Reply:
Spoke about this offline - Xin will implement after merge